default.yml

```diff
@@ -230,3 +210,10 @@ Rails/FindBy:
   Enabled: true
   Include:
     - app/**/*.rb
+
+# RSpec
+RSpec/AlignLeftLetBrace:
+  Enabled: true
+
+RSpec/AlignRightLetBrace:
+  Enabled: false
```

Comment:
This allows for:

```ruby
let(:my_short_name)      { "Tom" }
let(:my_much_longer_name { "Tom Longer Name Brooks" }
```

Reply:
This one I am impartial on, but tend to prefer they are aligned left.

Reply:
👍

Reply:
I agree. The right aligned brace is odd to me.